src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java

```diff
@@ -183,6 +186,22 @@ public void onCreate(Bundle paramBundle) {
       }
       return true;
     });
+
+    Preference selfReporting = this.findPreference("pref_self_reporting");
+    selfReporting.setOnPreferenceClickListener(((preference) -> {
```

Comment:
so they have to manually send them, from time to time :thinking:

Reply:
Yes, I wanted to start with something simple. We can always make things more complicated, and since for now this will only be implemented in Android (and not the other UIs), we can easily change the core API.